processor/workers.go

```diff
@@ -737,6 +767,15 @@ func fileProcessorWorker(input chan *FileJob, output chan *FileJob) {
 					duplicates.mux.Unlock()
 				}
 
+				if IgnoreMinifiedGenerate {
+					if res.Minified {
```

Comment:
Purely style preference, but I'd make this `IgnoreMinifiedGenerate && res.Minified` over the nested condition.

Reply:
100% agree with you. Will refactor.